challenge.js

```diff
@@ -42,7 +43,7 @@
  */
 
 // ANSWER
-
+A > D > E
```

Comment:
Great. Now, if you use the fact that you know their tag names, you can write this as `body > div > p`. 1